.github/workflows/jira-issue-transfer.yml

```diff
@@ -16,12 +16,13 @@
 name: Jira Issue Transfer
 
 on:
-  issues:
+  label:
     types: 
-      - opened
+      - edited
 jobs:
   build:
     runs-on: self-hosted
+    if: github.event.label.name == 'bug' && github.event.label.name == 'Ticketed'
```

Comment:
Would this ever actually be true? This looks like the logic requires the field to be two different values at the same time

Reply:
Updated commit - feel free to review